libs/SalesforceSDKCore/SalesforceSDKCore/Classes/SmartStore/SFSmartStoreDatabaseManager.h

```diff
@@ -46,6 +46,8 @@ extern NSString * const kSFSmartStoreDbErrorDomain;
  */
 + (SFSmartStoreDatabaseManager *)sharedManagerForUser:(SFUserAccount *)user;
 
++ (SFSmartStoreDatabaseManager *)sharedGlobalManager;
```

Comment:
Shouldn't this be ```(SFSmartStoreDatabaseManager *)sharedGlobalManager:(NSString*)dbName;```? Will the same manager be used across multiple global stores?